src/routes/cards.js

```diff
@@ -54,6 +54,7 @@ exports.plugin = {
               right: Joi.string().allow('', null).max(SURVEY_LABEL_MAX_LENGTH),
               left: Joi.string().allow('', null).max(SURVEY_LABEL_MAX_LENGTH),
             }),
+            areAnswersCommutative: Joi.boolean(),
```

Comment:
au lieu du prehandler tu peux peut-etre mettre ici
`areAnswersCommutative: Joi.boolean().when('template', { is: !FILL_THE_GAP, then: Joi.forbidden() }),`

Reply:
outdated